server/controllers/finance/reports/creditors/aged.handlebars

```diff
@@ -0,0 +1,77 @@
+{{> head title="REPORT.AGED_SUPPLIERS"}}
```

Comment:
You are mixing "Aged Suppliers" and "Aged Creditors" a lot.  I propose you stick to the name "Aged Creditors", to avoid confusing in file names and variable names.